cli/run_test.go

```diff
@@ -1011,16 +1008,10 @@ func Test0Steps1Workflows(t *testing.T) {
 	require.Equal(t, 0, len(buildRunResults.FailedSteps))
 	require.Equal(t, 0, len(buildRunResults.FailedSkippableSteps))
 	require.Equal(t, 0, len(buildRunResults.SkippedSteps))
-
-	require.Equal(t, "0", os.Getenv("BITRISE_BUILD_STATUS"))
```

Comment:
Build status env tests are moved to `_tests/integration/workflow_run_envs_test.go`